.heroku/release.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash 
+set -e
+
+if [ -n "$CTC_URL" ] ; then
+     # Run migration surrounded by CTC start/stop
+     python .heroku/check_change_traffic_control.py
+else
+     # Simply run the migration
+     python manage.py migrate --noinput
+     python .heroku/check_change_traffic_control.py
```

Comment:
The point of the if/else is that we don't want to call check_change_traffic_control.py if there's no CTC_URL

Reply:
Sorry I did this for my local testing.